src/wallet/wallet.cpp

```diff
@@ -4755,6 +4759,13 @@ bool CWallet::AutoBackupWallet(const fs::path& wallet_path, bilingual_str& error
 
     return true;
 }
+#elif USE_SQLITE
+bool CWallet::AutoBackupWallet(const fs::path& wallet_path, bilingual_str& error_string, std::vector<bilingual_str>& warnings)
+{
+    WalletLogPrintf("Automatic wallet backups are not supported with SQLite!\n");
```

Comment:
`AutoBackupWallet` leverages logic specific to Berkeley rather than using the database abstraction ([source](https://github.com/dashpay/dash/pull/5072/files/5a621fd991eb45abea0d92c568db3f233cea14aa#diff-1f2db0e4d5c12d109c7f0962333c245b49b696cb39ff432da048e9d6c08944d8L4692-L4696)). Resolving that is something to be done in the future. I tried doing a minimal disable but it breaks multiple tests, it seems like a clean substitution with a fast-fail is least problematic way of accounting for this.

Reply:
ok, I guess it's fine for now